test/transformers/babel.test.ts

```diff
@@ -40,4 +40,37 @@ $: bar = foo?.b ?? 120
       $: bar = (_foo$b = foo == null ? void 0 : foo.b) != null ? _foo$b : 120;</script>"
     `);
   });
+
+  it('should not transpile modules with preset-env', async () => {
```

Comment:
It still transpiles them, just not to CJS, according to the snapshot. I suggest to alter the test description a little

```suggestion
  it('should not transpile modules to CJS with preset-env', async () => {
```

Reply:
Hm, if we take a look at the snapshot then we have this:
```js
// input
import foo from './foo'

// snapshot
import foo from './foo';
```

Looks pretty similar to me, so the given description seems to be accurate. Unless I'm missing what you are referring to.

Reply:
My semantic understanding of the sentence was that "modules" is related to the whole file itself, not the module import. With the suggestion I tried to clarify that. Maybe "should not transpile imports with preset-env" is better?

Reply:
Ah, I see what you mean here - I've probably written it like this since `modules` option in `@babel/preset-env` configured how import/export syntax is going to be transpiled.

So maybe smth like this would be appropriate?
```suggestion
  it('should not transpile import/export syntax with preset-env', async () => {
```

I'm fine either way, so just pick whichever you prefer :)